recipe/build_base.sh

```diff
@@ -1,22 +1,12 @@
 #!/bin/bash
 
-cat <<EOF > conda_mpl_config.cfg
-[directories]
-basedirlist = $PREFIX
+set -ex
 
-[packages]
-tests = False
-toolkit_tests = False
-sample_data = False
+export AR=$GCC_AR
 
-[libs]
-system_freetype = True
+export MESON_ARGS="${MESON_ARGS} --buildtype=release -Dsystem-freetype=true -Dsystem-qhull=true"
```

Comment:
It looks like the source for these is [still getting downloaded]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963536&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951 )

Do we need to do something more to make sure the Conda packages are used and these are not downloaded?

```
Downloading freetype-2.6.1 source from https://download.savannah.gnu.org/releases/freetype/freetype-old/freetype-2.6.1.tar.gz

Executing subproject freetype-2.6.1 

freetype-2.6.1| Project name: freetype2
freetype-2.6.1| Project version: 2.6.1
freetype-2.6.1| C compiler for the host machine: /home/conda/feedstock_root/build_artifacts/matplotlib-suite_1719453551626/_build_env/bin/x86_64-conda-linux-gnu-cc (gcc 12.3.0 "x86_64-conda-linux-gnu-cc (conda-forge gcc 12.3.0-13) 12.3.0")
freetype-2.6.1| C linker for the host machine: /home/conda/feedstock_root/build_artifacts/matplotlib-suite_1719453551626/_build_env/bin/x86_64-conda-linux-gnu-cc ld.bfd 2.40
freetype-2.6.1| Has header "unistd.h" : YES
freetype-2.6.1| Has header "fcntl.h" : YES
freetype-2.6.1| Has header "stdint.h" : YES
freetype-2.6.1| Configuring ftconfig.h using configuration
freetype-2.6.1| Configuring ftoption.h using configuration
freetype-2.6.1| Build targets in project: 3
freetype-2.6.1| Subproject freetype-2.6.1 finished.

Downloading qhull source from https://github.com/qhull/qhull/archive/v8.0.2/qhull-8.0.2.tar.gz

Executing subproject qhull 

qhull| Project name: qhull
qhull| Project version: 8.0.2
qhull| C compiler for the host machine: /home/conda/feedstock_root/build_artifacts/matplotlib-suite_1719453551626/_build_env/bin/x86_64-conda-linux-gnu-cc (gcc 12.3.0 "x86_64-conda-linux-gnu-cc (conda-forge gcc 12.3.0-13) 12.3.0")
qhull| C linker for the host machine: /home/conda/feedstock_root/build_artifacts/matplotlib-suite_1719453551626/_build_env/bin/x86_64-conda-linux-gnu-cc ld.bfd 2.40
qhull| Build targets in project: 4
qhull| Subproject qhull finished.
```

Reply:
I don't think meson-python reads `$MESON_ARGS`, and certainly Meson doesn't; these need to be passed through [options through `build`](https://meson-python.readthedocs.io/en/latest/how-to-guides/config-settings.html). The `meson setup builddir` is not doing anything since you didn't tell [meson-python to use that directory](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963536&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=55f066f8-9d52-54c8-1ae3-a2d9002ff304&l=918).

Reply:
Am no longer seeing this error in recent builds (for [example]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963561&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=55f066f8-9d52-54c8-1ae3-a2d9002ff304 )). So think it is now doing the right thing. Probably had a flag issue that has since been corrected